### DISCUSSION TITLE
I have a problem when I use UPFD in datasets.May be a bug？

### DISCUSSION THREAD

**Discussion by Leon-377** [pyg-team/pytorch_geometric#8408] [Q&A]
I have a problem when I use UPFD in datasets.
It successfully downloaded and unzipped and converted to Data data, where the data.x data is normal, but there is a problem with data.edge_index. Specifically, the data.edge_index of the first dataset is empty with `edge_index=[2, 0]`, `tensor([], size=(2, 0), dtype=torch.int64)`.
The link when downloading the dataset should be fine https://docs.google.com/uc?export=download&id=1VskhAQ92PrT4sWEKQ2v2-AJhEcpp4A81&confirm=t.
This is not the same as past usage (about 6 months ago), which worked fine in the past.
Is it a problem with the new version please?

**Comment by Leon-377** [comment]
The link to the original article was provided by PyG.
The link to the dataset provided by the original author is https://drive.google.com/drive/folders/1OslTX91kLEYIi2WBnwuFtXsVz5SS_XeR
Since it can be downloaded and the data.x is working, I think the link should be fine.

**Comment by rusty1s** [comment]
Looks like we introduced a bug there in PyG 2.4, sorry :( Will be fixed in https://github.com/pyg-team/pytorch_geometric/pull/8413.
